growth/too/tasks/obs.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from astropy import time
+import astropy.units as u
+from pyvo.dal import TAPService
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_obs',)
+
+
+def get_tap_client():
+    url = 'https://irsa.ipac.caltech.edu/TAP'
+    return TAPService(url)
+
+
+@celery.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_obs(start_time=None, end_time=None):
+
+    if start_time is None:
+        start_time = time.Time.now() - time.TimeDelta(1.0*u.day)
+    if end_time is None:
+        end_time = time.Time.now()
+
+    obstable = get_tap_client().search("""
+    SELECT field,ccdid,qid,fid,expid,obsjd,exptime,seeing,airmass,maglimit
+    FROM ztf.ztf_current_meta_sci WHERE (obsjd BETWEEN %s AND %s)
+    """ % (start_time.jd, end_time.jd)).to_table()
+    obstable = obstable.filled()
+    obstable['subfield_id'] = (obstable['ccdid']-1)*4 + obstable['qid']
```

Comment:
I noticed that there was also an `rcid` column with the description `Readout-channel ID (0..63)`. Could we use that instead?